google-cloud-storage/src/main/resources/META-INF/native-image/com/google/cloud/storage/proxy-config.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+    "interfaces":["com.google.cloud.storage.spi.v1.StorageRpc"]
+  }
+]
```

Comment:
Is it possible to move this file to `src/test/resources/...` since it's only needed for tests?

Reply:
Great, thank you!